client/coral-framework/loaders/plugins-loader.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Executes `source` to retrieve plugins configuration
```

Comment:
The output of the `loader` before this one, which currently is the `json-loader`.